modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/forms/DDMFormHandler.js

```diff
@@ -28,9 +28,11 @@ class DDMFormHandler {
 	}
 
 	_attachFormListener() {
-		this.DDMFormInstance.on('fieldEdited', (field) => {
-			this.fields = updateFields(this.fields, field);
-			this.checkCPInstance();
+		this.DDMFormInstance.unstable_onEvent(({type, payload}) => {
+			if (type === 'fieldEdited') {
+				this.fields = updateFields(this.fields, payload);
+				this.checkCPInstance();
+			}
```

Comment:
Hey @gianmarcobrunialti I made some changes here to listen to the internal events of the Forms but I can't test if this is working well, the tests I did were manual, checking the instance of the form and adding the callback. Is there anything else I can test to see this file working?

I can't see this file when I'm on the page to add a product image, I'm following the steps in COMMERCE-5478.

This API should also change to something more stable and a more consistent API in the future, so I am adding `unstable` here.

Reply:
Hey @matuzalemsteles! I'll test this out myself to see if our listeners work after this change 👍 Updating you in some minutes!